src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let partnerMessage;
+
+  if (person.isMarried) {
+    if (person.sex === 'm') {
```

Comment:
```suggestion
  if (isMarried) {
    if (sex === MAN) {
```
Consider creating a constant with descriptive name here